src/main/java/UI/CreateListingPage.java

```diff
@@ -1,84 +1,131 @@
 package UI;
+import Main.Main;
 import forms.CreateListingForm;
+import forms.LoginForm;
 import useCase.createListing.ListingInteractor;
+import useCase.createListing.ListingResponseModel;
 
+import javax.imageio.ImageIO;
 import javax.swing.*;
 import java.awt.*;
 import java.awt.event.ActionEvent;
 import java.awt.event.ActionListener;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
 import java.text.DecimalFormat;
 import java.util.List;
 import java.util.ArrayList;
```

Comment:
There are a lot of imports here. Do you need them all? It's a good idea to clean up the files before pushing to main.